HeavyFlavorAnalysis/SpecificDecay/plugins/BPHHistoSpecificDecay.h

```diff
@@ -0,0 +1,95 @@
+#ifndef TestBaseNtuple_h
+#define TestBaseNtuple_h
+
+#include "HeavyFlavorAnalysis/RecoDecay/interface/BPHAnalyzerTokenWrapper.h"
+#include "DataFormats/PatCandidates/interface/CompositeCandidate.h"
+
+#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "DataFormats/Common/interface/Ref.h"
+
+#include <string>
+
+class TH1F;
+class TVector3;
+
+namespace reco {
+  class Candidate;
+  class Vertex;
+}
+
+class BPHHistoSpecificDecay: public BPHAnalyzerWrapper<edm::EDAnalyzer> {
```

Comment:
edm::one::EDAnalyzer is more appropriate instead of the legacy module edm::EDAnalyzer.